docs/content/users/extend/additional-services.md

```diff
@@ -105,18 +105,25 @@ If you have add-ons that were installed before v1.22, update them with `ddev get
 
 ## Adding Custom Configuration to an Add-on
 
-Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis-7`](https://github.com/ddev/ddev-redis-7) the `image` is set to `image: redis:7.2-alpine`. If you wanted to change this to use `7.0-alpine` instead, you would have two choices:
+Sometimes it's necessary to add custom configuration to an add-on. For example, in [`ddev-redis`](https://github.com/ddev/ddev-redis) the `image` is set to `image: redis:${REDIS_VERSION:-6-bullseye}`. If you wanted to change this to use `7.0-alpine` instead, you would have three choices:
 
-1. Remove the `#ddev-generated` line in `docker-compose.redis-7.yaml` and edit the relevant line.
-2. Add a `.ddev/docker-compose.redis-7_extra.yaml` with the contents:
+1. With DDEV v1.23.4+, the override can be done using a special `.ddev/.env.*` file, where `*` is the name of the add-on, in this case `.ddev/.env.redis`:
 
-```yaml
-  services:
-    redis:
-      image: redis:7.0-alpine
-```
+    ```bash
+    ddev get ddev/ddev-redis --environment="REDIS_VERSION=7.0-alpine"
+    ddev restart
+    ```
+
+2. Remove the `#ddev-generated` line in `.ddev/docker-compose.redis.yaml` and edit the relevant line.
```

Comment:
Why don't we just stop recommending this, since we have better ways? The general docs explain this a few places, but it's not even needed here.

```suggestion
```